packages/angular_devkit/build_angular/src/browser/index.ts

```diff
@@ -668,7 +668,7 @@ export function buildWebpackBrowser(
                       const { content, warnings, errors } = await indexHtmlGenerator.process({
                         baseHref: getLocaleBaseHref(i18n, locale) || options.baseHref,
                         // i18nLocale is used when Ivy is disabled
-                        lang: locale || options.i18nLocale,
+                        lang: locale || undefined,
```

Comment:
Is the undefined needed here?

Reply:
It is needed because when it’s locale empty string, a lang attribute is added with a an empty string.

Maybe the index generation shouldn’t generate the lang attribute if lang is an empty string.

Reply:
ok.  This works for now.  We can look at that in the future.